packages/relay/src/QueryRenderer.js

```diff
@@ -31,7 +34,7 @@ export default class QueryRenderer extends React.Component<Props> {
 
   renderRelayContainer = ({ error, props }: RendererResponse) => {
     if (error) {
-      if (error.message === 'Network request failed') {
+      if (errorMessages.includes(error.message)) {
         // See: https://github.com/github/fetch/blob/fcc4e1b48cfb5a2b1625fcd6eac06d954b00ccb6/fetch.js#L438-L444
```

Comment:
I think this comment should be moved because it's an explanation of why this exact string _"Network request failed"_ is used here.